*** /sw/share/octave/3.0.0/m/plot/drawnow.m	2008-01-07 16:28:12.000000000 +0900
--- drawnow.m	2008-01-09 23:00:26.000000000 +0900
***************
*** 208,214 ****
      ## should not be interpreted in that case.
      if (compare_versions (__gnuplot_version__ (), "4.0", ">"))
        enhanced_terminals = {"aqua", "dumb", "png", "jpeg", "gif", "pm", ...
! 	                    "windows", "wxt", "svg", "postscript", "x11"};
      else 
        enhanced_terminals = {"x11", "postscript"};
      endif
--- 208,214 ----
      ## should not be interpreted in that case.
      if (compare_versions (__gnuplot_version__ (), "4.0", ">"))
        enhanced_terminals = {"aqua", "dumb", "png", "jpeg", "gif", "pm", ...
! 	                    "windows", "wxt", "svg", "postscript", "x11", "pdf"};
      else 
        enhanced_terminals = {"x11", "postscript"};
      endif
